03-interactive-site/README.md

```diff
@@ -1,57 +1,55 @@
+2. Disponibilizar um site para apresentar informações \(texto, imagens, links, etc.\) na web, usando `html` e `css`
+3. Dar interatividade ao seu site da web através da programação com `JavaScript`
 
-### Público alvo
+## Público alvo
 
 Este curso é destinado às estudantes da Laboratoria. É o segundo curso que é tomado como parte do _bootcamp_.
 
-### Pré-requisitos
+## Pré-requisitos
 
 Para fazer este curso você deve ter concluído o curso `Programação Básica com JavaScript` da Laboratoria. Supõe-se que os alunos saibam como criar programas básicos, fazendo uso de: variáveis, funções, loops, condicionais, vetores e objetos.
 
-### Objetivo Geral
+## Objetivo Geral
 
 O objetivo deste curso é aplicar o que você sabe sobre programação em site na web. Neste curso, você criará sites da Web com `html` e `css` e você dará interação através de programação com `JavaScript`.
 
-### Você aprenderá
+## Você aprenderá
 
-Criar uma página web estática usando as principais marcações `html` e estilizá-la com `css`  
-O que é o Document Object Model \(DOM\) e o Browser Object Model \(BOM\) e como utilizar JavaScript para para manipular o `html` e \`css\` do seu produto  
-O que é o `ux design` e por que é uma peça-chave para o desenvolvimento de um produto digital
+* Criar uma página web estática usando as principais marcações `html` e estilizá-la com `css`
+* O que é o Document Object Model \(DOM\) e o Browser Object Model \(BOM\) e como utilizar JavaScript para para manipular o `html` e \`css\` do seu produto
+* O que é o `ux design` e por que é uma peça-chave para o desenvolvimento de um produto digital
 
 Tags principais: `html`, `css properties`, `css selectors`, `dom`, `bom`, `events`, `ux design`.
 
 Tags Secundárias: `display`, `box model`, `positioning`, `layout`, `sketching`.
 
-### Duração
+## Duração
 
 150 horas aprox.
 
-### Produtos para desenvolvimento
+## Produtos a desenvolver
 
 Com este curso você desenvolverá seus primeiros produtos digitais. Você vai colocar em prática o conhecimento aprendido para criar os seguintes produtos da web:
 
-1. **Portfólio**: você criará sua página pessoal para mostrar suas informações profissional através da web.  
-2. **Jogo do Labirinto**: você criará seu primeiro jogo animado! O jogador deve mover o personagem principal usando as setas e encontrar a saída em um labirinto.  
+1. **Portfólio**: você criará sua página pessoal para mostrar suas informações profissional através da web.
+2. **Jogo do Labirinto**: você criará seu primeiro jogo animado! O jogador deve mover o personagem principal usando as setas e encontrar a saída em um labirinto.
 3. **Data Dashboard**: seu produto final será projetar e criar um site interativo, de média complexidade, mostrando os dados e o perfil de uma estudante da Laboratoria.
 
-### Ementa
+## Syllabus
 
 ### Unidade 01: [Layout da Web com HTML & CSS](https://github.com/Laboratoria/curricula-js/blob/03-interactive/03-interactive-site/00-html-and-css)
```

Comment:
The parser needs to know the unit's folder name. So, here the link should be `00-html-and-css`.